### PR TITLE
Added time, namespace label

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -133,10 +134,12 @@ def _do_sync(
     if not only_one_kind or only_solver_documents:
         _LOGGER.info("Syncing solver results")
         category = "solver"
-        _METRIC_SECONDS.labels(category=category).time()
+        start = time.time()
```
Comment (fridex) [inline app.py]: It would be better to use [monotonic time for this](https://docs.python.org/3/library/time.html#time.monotonic).

Comment (pacospace) [reply app.py]: Added thanks @fridex !

File: app.py [hunk 2]
```diff
@@ -117,6 +123,11 @@ def _do_sync(
             int(only_dependency_monkey_documents),
         )
     )
+    environment_varibale = "THOTH_NAMESPACE"
+    if os.getenv(environment_varibale):
+        namespace = os.getenv(environment_varibale)
```
Comment (fridex) [inline app.py]:
```python
environment_variable = os.getenv("THOTH_NAMESPACE")
if not environment_variable:
        _LOGGER.warning("Namespace variable not provided for %r", environment_varibale)
```

Note the `%r` to expand the warning message.

Comment (pacospace) [reply app.py]: Thanks!

**Pull request merged**